packages/main/src/BreadcrumbsItem.js

```diff
@@ -0,0 +1,108 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+
+/**
+* @public
+*/
+const metadata = {
+	tag: "ui5-breadcrumbs-item",
+	managedSlots: true,
```

Comment:
I think you don't need "managedSlots: true," here, probably copied.

Reply:
I added "managedSlots: true" to be able to react (**recalculate the number of overflowing links**) whenever the text of the link changes.

Do you have a better idea in mind?

Btw earlier I used a ResizeHandler on a content wrapper, but seemed like the less efficient solution, especially to monitor changes in the size of links that are currently in the overflow.